elementary/monitor/cli.py

```diff
@@ -454,6 +454,18 @@ def report(
     default=False,
     help="If set to true elementary report won't show data metrics for passed tests (this can improve report creation time).",
 )
+@click.option(
+    "--disable",
+    type=str,
+    default=None,
+    help='Disable functualities from the "send-report" command.\nCurrently only --disable html_attachment is supported.',
+)
+@click.option(
+    "--include",
+    type=str,
+    default=None,
+    help="Include additional information at the test results summary message.\nCurrently only --include descriptions is supported.",
+)
```

Comment:
Why don't we include descriptions by default? It's too detailed?

Reply:
Yes 🙂 (product definition) + It is padding the attachments (which has small size limit)